src/encoded/tests/data/inserts/quality_metric_rnaseq_madqc.json

```diff
@@ -0,0 +1,71 @@
+[
+    {
+        "uuid": "d0be03da-3700-4148-a225-3852422c121e",
+        "award": "1U01CA200059-01",
+        "lab": "4dn-dcic-lab",
+        "MAD QC": {
+          "one and two": {
+            "MAD of log ratios": 0.398,
+            "Pearson correlation": 0.949,
+            "SD of log ratios": 0.573,
+            "Spearman correlation": 0.929
+          },
+          "one and three": {
+            "MAD of log ratios": 0.361,
+            "Pearson correlation": 0.955,
+            "SD of log ratios": 0.54,
+            "Spearman correlation": 0.936
+          },
+          "one and four": {
+            "MAD of log ratios": 2.013,
+            "Pearson correlation": 0.427,
+            "SD of log ratios": 2.358,
+            "Spearman correlation": 0.401
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 spaces but found 12.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4828388)